src/main/java/com/netgrif/application/engine/workflow/service/TaskService.java

```diff
@@ -794,10 +794,10 @@ private Task createFromTransition(Transition transition, Case useCase) {
             task.setTransactionId(transaction.getStringId());
         }
 
-        resolveUserRef(task, useCase);
         Task savedTask = save(task);
 
         useCase.addTask(savedTask);
+        workflowService.resolveUserRef(useCase);
         useCase = workflowService.save(useCase);
```

Comment:
`resolveUserRef()` returns Case object and also saves changes calling `repository.save()`, consider saving the returned object or maybe calling `workflowService.save()` in `resolveUserRef` function

Reply:
Done

Reply:
Done? has this change been pushed? Because I can still see the problems described by @mazarijuraj

Reply:
Done, pushed unpushed commit.